src/components/ha-climate-range-control.html

```diff
@@ -0,0 +1,135 @@
+<link rel='import' href='../../bower_components/polymer/polymer.html'>
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+
+
+<dom-module id="ha-climate-range-control">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      /* local DOM styles go here */
+      :host {
+        @apply(--layout-flex);
+        @apply(--layout-horizontal);
+        @apply(--layout-justified);
+      }
+      .target-temperature {
+        @apply(--layout-self-center);
+        font-size: 200%;
+      }
+      .control-buttons {
+        font-size: 200%;
+        text-align: right;
+      }
+      paper-icon-button {
+        height: 48px;
+        width: 48px;
+      }
+    </style>
+
+    <!-- local DOM goes here -->
+    <div class="target-temperature">
```

Comment:
Because I couldn't get it to work right the other way and this is my first time messing with HASS UI or Polymer in general (oh, and I *hate* UI development)? I'm kinda throwing stuff at the wall and seeing what sticks. Maybe I should modify the original to work better in a side-by-side situation?